plugins/woocommerce-blocks/assets/js/blocks/product-filters/inner-blocks/stock-filter/block.json

```diff
@@ -1,53 +1,16 @@
 {
+	"$schema": "https://schemas.wp.org/trunk/block.json",
 	"name": "woocommerce/product-filter-stock-status",
 	"version": "1.0.0",
-	"title": "Filter Options",
-	"description": "Enable customers to filter the product collection by stock status.",
+	"title": "Stock (Experimental)",
+	"description": "Let shoppers filter products by choosing stock status.",
 	"category": "woocommerce",
-	"keywords": [
-		"WooCommerce",
-		"filter",
-		"stock"
-	],
-	"supports": {
-		"interactivity": true,
-		"html": false,
-		"inserter": false,
-		"color": {
-			"text": true,
-			"background": false
-		}
-	},
-	"attributes": {
-		"className": {
-			"type": "string",
-			"default": ""
-		},
-		"showCounts": {
-			"type": "boolean",
-			"default": false
-		},
-		"displayStyle": {
-			"type": "string",
-			"default": "list"
-		},
-		"selectType": {
-			"type": "string",
-			"default": "multiple"
-		},
-		"isPreview": {
-			"type": "boolean",
-			"default": false
-		}
```

Comment:
These attributes still belong to the Stock filter in the new structure, you can check the Attribute block for reference.

Reply:
Oh, checking the Figma mockup, it looks like that I missed the list support as well. I'm going to work on it!

Reply:
The stock filter can switch between List or Chips block, like Attribute.

Reply:
Added the support for List block with 1083547.